mmdet/datasets/transforms/transforms.py

```diff
@@ -3005,3 +3005,20 @@ def __repr__(self):
         repr_str += f'mask_border_value={self.mask_border_value}, '
         repr_str += f'seg_ignore_label={self.seg_ignore_label})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class CropBoxes(BaseTransform):
+    """Make sure boxes are within the image range."""
+
+    @autocast_box_type()
+    def transform(self, results: dict) -> dict:
```

Comment:
Is this pipeline necessary? Many pipeline will clip box boundary when setting `bbox_clip_border=True`.

If necessary, it's better to use results['gt_bboxes'].clip_(results['img_shape']). The gt_bboxes will be cropped in-place.

Reply:
I found the option to crop the label box in `Resize`, so it's redundant for me to use `CropBoxes` before using `Resize`. I have removed this code from PR.